src/components/List.js

```diff
@@ -183,42 +183,46 @@ const List = ({ token }) => {
                         item.itemName
                       } ready to purchase ${backgroundColor(item)}`}
                     >
-                      <Checkbox
-                        type="checkbox"
-                        icon={
-                          <CircleUnchecked
-                            style={{
-                              color: checkboxColor[backgroundColor(item)],
-                            }}
+                      <div className="flex-container">
+                        <div>
+                          <Checkbox
+                            type="checkbox"
+                            icon={
+                              <CircleUnchecked
+                                style={{
+                                  color: checkboxColor[backgroundColor(item)],
+                                }}
+                              />
+                            }
+                            checkedIcon={
+                              <CircleCheckedFilled
+                                style={{
+                                  color: checkboxColor[backgroundColor(item)],
+                                }}
+                              />
+                            }
+                            aria-label="purchased-checkbox"
+                            name={item.itemName}
+                            id={item.id}
+                            onChange={handleCheckbox}
+                            checked={checkPurchasedDate(item.lastPurchased)}
+                            disabled={checkPurchasedDate(item.lastPurchased)}
                           />
-                        }
-                        checkedIcon={
-                          <CircleCheckedFilled
-                            style={{
-                              color: checkboxColor[backgroundColor(item)],
-                            }}
-                          />
-                        }
-                        aria-label="purchased-checkbox"
-                        name={item.itemName}
-                        id={item.id}
-                        onChange={handleCheckbox}
-                        checked={checkPurchasedDate(item.lastPurchased)}
-                        disabled={checkPurchasedDate(item.lastPurchased)}
-                      />
-                      {item.itemName}{' '}
-                      <button onClick={() => confirmDelete(item)}>
-                        Delete
-                      </button>
-                      {item.numberOfPurchases > 0 ? (
+                          {item.itemName}{' '}
+                        </div>
+                        <button onClick={() => confirmDelete(item)}>
+                          Delete
+                        </button>
+                      </div>
+                      {/* {item.numberOfPurchases > 0 ? (
                         <p>
```

Comment:
We may use these comments in the future so commenting out for now.

Reply:
Not a bad idea to keep stuff like that around 👍 
I'd recommend adding a comment explaining that you might use it later. 
Otherwise, someone else might delete it on accident, thinking it was leftover by mistake 🙈